### PR TITLE
Added HTML, CSS resource/course hindi playlist link

### PR DESCRIPTION
**Pull request by MayankParkar**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Great resource for MERN stack which is on going along with projects at regular interval to practice and revise.
https://www.youtube.com/watch?v=Vi9bxu-M-ag&list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD&pp=iAQB

### Why is this valuable (or not)?
It is value and very good resource for HTML and CSS along with projects as per the topics.

### How do we know it's really free?
It is free playlist on youtube made by Love Babbar. He also made a DSA placement series which is already their in this repo.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -307,6 +307,7 @@
 
 ### HTML and CSS
 
+* [Complete Web Dev using mern stack Love Babbar](https://www.youtube.com/watch?v=Vi9bxu-M-ag&list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD&pp=iAQB) (Love Babbar)
```
Comment (eshellman) [inline courses/free-courses-hi.md:310]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by eshellman** [comment]
also, please keep changes on this PR. don't close it and start a new one. It's not respectful to the reviewers because it erases the reviews.

**Comment by MayankParkar** [comment]
okay, i'll keep that in mind. Thank you

File: courses/free-courses-hi.md (hunk 1 above)
Comment (MayankParkar) [reply courses/free-courses-hi.md:310]: I have mentioned the playlist link and the creator name that is "Love babbar"

Comment (eshellman) [reply courses/free-courses-hi.md:310]: thiis is the playlist: https://www.youtube.com/playlist?list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD

Comment (MayankParkar) [reply courses/free-courses-hi.md]: Changed and saved the new link

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -307,6 +307,7 @@
 
 ### HTML and CSS
 
+* [Complete Web Dev using mern stack Love Babbar](https://www.youtube.com/playlist?list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD) (Love Babbar)
```
Comment (eshellman) [inline courses/free-courses-hi.md:310]:
```suggestion
* [Complete Web Dev using mern stack Love Babbar](https://www.youtube.com/playlist?list=PLDzeHZWIZsTo0wSBcg4-NMIbC0L8evLrD) - Love Babbar
```

File: courses/free-courses-hi.md (hunk 1 above)
Comment (MayankParkar) [reply courses/free-courses-hi.md]: Changes done. - Love babbar

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**